docs/issuing-assets/how-to-issue-an-asset.mdx

```diff
@@ -72,21 +204,82 @@ const transaction = new StellarSdk.TransactionBuilder(account, {
   networkPassphrase: StellarSdk.Networks.TESTNET,
 })
   // The `changeTrust` operation creates (or alters) a trustline
-  // The `limit` parameter below is optional
   .addOperation(
     StellarSdk.Operation.changeTrust({
       asset: astroDollar,
-      limit: "1000",
+      limit: "1000", // optional
       source: distributorKeypair.publicKey(),
     }),
-  );
+  )
+  .setTimeout(100)
+  .build();
+```
+
+```java
+Server server = new Server("https://horizon-testnet.stellar.org");
+AccountResponse distributorAccount = server.accounts().account(distributorKeypair.getAccountId());
+
+Transaction transaction = new Transaction.Builder(distributorAccount, Network.TESTNET)
+  .addOperation(
+    new ChangeTrustOperation.Builder(astroDollar, "1000")
+    .setSourceAccount(distributorKeypair.getAccountId())
+    .build()
+  )
+  .setTimeout(100)
+  .build();
+```
+
+```go
+import (
+  "github.com/stellar/go/clients/horizonclient"
+  "github.com/stellar/go/keypair"
+  "github.com/stellar/go/network"
+  "github.com/stellar/go/txnbuild"
+)
+
+client := horizonclient.DefaultTestNetClient
+distributorAccountRequest := horizonclient.AccountRequest{AccountID: distributorKeypair.Address()}
+distributorAccount, _ := client.AccountDetail(distributorAccountRequest)
+
+transaction, _ := txnbuild.NewTransaction(
+  txnbuild.TransactionParams{
+    SourceAccount:        &distributorAccount,
+    IncrementSequenceNum: true,
+    BaseFee:              txnbuild.MinBaseFee,
+    Operations: []txnbuild.Operation{
+      &txnbuild.ChangeTrust{
+        Line:  astroDollar,
+        Limit: "1000",
+      },
+    },
+    Timebounds: txnbuild.NewInfiniteTimeout(),
+  },
+)
+```
+
+```python
+from stellar_sdk import Keypair, Asset, Network, Server, TransactionBuilder
+
+server = Server("https://horizon-testnet.stellar.org")
+distributor_account = server.load_account(distributor_keypair.public_key)
+
+transaction = (
+  TransactionBuilder(
+    source_account=distributor_account,
+    network_passphrase=Network.TESTNET_NETWORK_PASSPHRASE,
+    base_fee=100
+  )
+  .append_change_trust_op(asset=astro_dollar, limit="1000")
+  .set_timeout(100)
+  .build()
+)
 ```
 
 </CodeExample>
 
-## 4. Make a payment from issuing to distribution account, issuing the asset
+### Issuer payment to distributor
 
-The payment operation is what actually issues (or mints) the asset.
+The payment operation is the most popular tool to actually issue (or mint) your asset. It creates the amount of an asset specified, up to [the maximum 64-bit integer](../learn/fundamentals/stellar-data-structures/assets.mdx#amount-precision). Relevantly, you do not need to scale up the issuing amount of your asset by the XDR minimum increment.
```

Comment:
In Solidity e.g., you may adjust the number of tokens issued by a decimal scaling factor:
```javascript
pragma solidity ^0.8.0;

import "@openzeppelin/contracts/token/ERC20/ERC20.sol";

contract MyToken is ERC20 {
    constructor(uint256 initialSupply) ERC20("MyToken", "MTK") {
        // The initial supply is multiplied by 10^18 to account for 18 decimal places
        _mint(msg.sender, initialSupply * 10 ** uint(decimals()));
    }
    
    function decimals() public view virtual override returns (uint8) {
        return 18;
    }
}
```

I thought it might be relevant to note that you don't need to scale by Stellar's [minimum increment](https://developers.stellar.org/docs/learn/fundamentals/stellar-data-structures/assets#amount-precision).

What do you think?

Reply:
Oooh, so you're basically saying something like "unlike Solidity, the decimal precision is already baked-in when using the `payment` operation"? I like it. Sounds like useful information for someone coming from EVM chains.

(I'm a dummy at Solidity, so I might be completely missing the point lol)